src/service/logService.js

```
@@ -0,0 +1,14 @@
+import axiosInstance from "./axiosInstance";
+
+class LogService {
+    static async getLogs(){
+        try {
+            const response = await axiosInstance.get(`/log/logs`);
+            return response.data;
+        }catch(error){
+            throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)